fuel-block-producer/src/block_producer/transaction_selector.rs

```diff
@@ -0,0 +1,173 @@
+use std::{
```

Comment:
This file was moved to the TxPool adapter, as this logic should eventually move to txpool itself. I think this may have re-appeared due to a merge conflict.

Reply:
Removed.